parser.go

```diff
@@ -362,9 +362,11 @@ func (g *Generator) genSchemaForType(fType reflect.Type) SchemaObj {
 	case reflect.String:
 		smObj = g.genSchemaForCommonName("string")
 	case reflect.Array, reflect.Slice:
-		smObj.Type = "array"
-		itemSchema := g.genSchemaForType(fType.Elem())
-		smObj.Items = &itemSchema
+		if (fType.String() != "json.RawMessage") {
```

Comment:
I think it would be better to declare a global variable for the reflect Type of `json.RawMessage` somewhere, then use it to compare with current type.

```go
var jsonRawMsgType = reflect.TypeOf((*json.RawMessage)(nil)).Elem()
....
		if fType != jsonRawMsgType {
			smObj.Type = "array"
			itemSchema := g.genSchemaForType(fType.Elem())
			smObj.Items = &itemSchema
		}
...
```